common/templates/start/_adoc-file

```diff
@@ -2,81 +2,117 @@
+//   - Keep in mind that this is a "getting started" guide.
+//   - Write to the audience you are trying to reach.
+//   - Add or remove sections and subsections as needed.
 // 
 // = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
 
 
 // = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
-// Document attributes and variables
+// DOCUMENT ATTRIBUTES AND VARIABLES
 //
 // NOTES:
-// 1. Update variables below and adjust docbook file accordingly.
-// 2. Comment out any variables/attributes not used.
-// 3. Follow the pattern to include additional variables.
+// 1. Define and use document attributes or variables in this file.
+// 2. Update the docinfo.xml file if needed.
 //
 // = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
 
+
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-// organization - do NOT modify
+// ORGANIZATION
+//   Do NOT modify this section.
 // -
 :trd: Technical Reference Documentation
 :type: Getting Started
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// DOCUMENT REVISION DATE
+//-
+:revision-date: YYYY-MM-DD
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// DOCUMENT TITLE AND SUBTITLE
+//   Provide concise but descriptive title and subtitle.
+// -
+:title: (<75 characters) Concisely identify the guide (e.g., "Kubeflow with Rancher")
+:subtitle: (<75 characters) Use for clarification (e.g., "Deploying Kubeflow onto an RKE2 Cluster with Rancher")
+
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// TECHNICAL COMPONENTS
+//   Identify the technical components featured in the guide.
+//   Define variables for various product information, such as
+//     product names (long and short), versions, websites, etc.
+//   For additional components, replicate the pattern (e.g.,
+//     scomp2, scomp3-version, pcomp2-website, etc.)
+//   Remove or comment out unneeded variables.
+// -
+// SUSE Components
+// -
+:scomp1-provider: SUSE
+:scomp1: Primary SUSE product short name (e.g., SLES, Rancher Prime)
+:scomp1-full: Primary SUSE product full name (e.g., SUSE Linux Enterprise Server, Rancher Prime by SUSE)
+:scomp1-version: Primary SUSE product relevant version (e.g., '15', '15SP5', '2.7') or versions (e.g., '15SP4, 15SP5', '15SP3+', '2.6+', '2.x')
+:scomp1-website: Primary SUSE product website URL
+:scomp1-docs: Primary SUSE product documentation URL
+
+// -
+// Partner/Project Components
+// -
+:pcomp1-provider: Partner/Project name
+:pcomp1: Partner/Project component short name
+:pcomp1-full: Partner/Project component full name
+:pcomp1-version: Partner/Project component version or versions
+:pcomp1-page: Partner/Project component website URL
```

Comment:
Here you name it "pcomp1-page", but some lines above you have "scomp1-website". Are both semantically the same? If yes, I would suggest to use _one_ suffix to stay consistent.

Reply:
does the `p` stand for partner, and the `s` for SUSE? That would make sense. Only question would be: do we need to split it down in that much detail and differ between partner and SUSE components? If this is important, fine for me - just asking :-).

Reply:
You are correct.
There is no semantic difference.